website/content/intro/vs/terraform.mdx

```diff
@@ -33,4 +33,4 @@ Vagrant is for development environments. Terraform is for more general
 infrastructure management.
 
 [hashicorp]: https://www.hashicorp.com
-[terraform]: https://www.terraform.io
```

Comment:
We should leave this link to the .io site homepage no?

Reply:
Sharing an update for posterity: we're chatting about this in Slack and confirming how we want to handle these links

Reply:
Update since [dev-portal#1549](https://github.com/hashicorp/dev-portal/pull/1549) was merged:

- Links to .io home pages are not rewritten
- `/` links are also not rewritten (to enable linking to Dev Dot home page)